Description: Fix keepassxc profile
 We enabled /etc/alternatives for nvidia/bumblebee LibGL support,
 and disabled memory execution protection as
 it does not seem to properly unlock the db with otherwise.
Author: Lorenzo "Palinuro" Faletra <palinuro@parrotsec.org>
Last-Update: 2018-05-25

--- firejail-0.9.54.orig/etc/keepassxc.profile
+++ firejail-0.9.54/etc/keepassxc.profile
@@ -38,10 +39,10 @@ shell none

 private-bin keepassxc
 private-dev
-private-etc fonts,ld.so.cache,machine-id
+private-etc fonts,ld.so.cache,machine-id,alternatives
 private-tmp

-# 2.2.4 crashes on database open
-#memory-deny-write-execute
+# 2.3.1 does not seem to properly unlock the kdbx file
+# memory-deny-write-execute
 noexec ${HOME}
 noexec /tmp
